lib/src/json_schema/validator.dart

```diff
@@ -61,6 +61,45 @@ class Instance {
 
   @override
   toString() => data.toString();
+
+  @override
+  bool operator ==(Object other) => other is Instance && this.path == other.path;
+
+  @override
+  int get hashCode => this.path.hashCode;
```

Comment:
I added this overrides to make `Set<Instance>` work. I think using the `path` is the proper way to check for identity of these identifiers.

Reply:
Could there be more than one path that arrives at the same identifier?

Reply:
I suppose yes, but those would be separate `Instance` objects. So this is probably fine.